### PR TITLE
docs: adding End Users section to ics29 docs

### PR DESCRIPTION
**Pull request by seantking**
## Description

partially closes: #1408

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: docs/middleware/ics29-fee/end-users.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- CLI Users who want to manually incentivize IBC packets
+- Client Developers
+
+
+The Fee Middleware module allows end users to add a 'tip' to each IBC packet which will incentivize relayer operators to relay packets between chains. GRPC endpoints are exposed for client developers as well as a simple CLI for manually incentivizing IBC packets.
+
+## CLI Users
+
+For an in depth guide on how to use the ICS29 Fee module using the CLI please take a look at the [wiki](https://github.com/cosmos/ibc-go/wiki/Fee-enabled-fungible-token-transfers#asynchronous-incentivization-of-a-fungible-token-transfer) on the `ibc-go` repo.
```
Comment (seantking) [inline docs/middleware/ics29-fee/end-users.md:25]: I think we can keep this section simple as there's no point in re-creating the wheel. Most people won't use the CLI anyway.

File: docs/middleware/ics29-fee/end-users.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!--
+order: 6
+-->
+
+# For End Users
```
Comment (crodriguezvega) [inline docs/middleware/ics29-fee/end-users.md:5]:
```suggestion
# For end users
```
```suggestion
# For End Users
```

File: docs/middleware/ics29-fee/end-users.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Learn how to incentivize IBC packets using the ICS29-Fee module. {synopsis}
+
+## Pre-requisite Readings
```
Comment (crodriguezvega) [inline docs/middleware/ics29-fee/end-users.md:9]:
```suggestion
## Pre-requisite readings
```
```suggestion
## Pre-requisite Readings
```

File: docs/middleware/ics29-fee/end-users.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+* [Fee Middleware](overview.md) {prereq}
+
+## Summary
+
+Different types of end users:
+
+- CLI Users who want to manually incentivize IBC packets
```
Comment (crodriguezvega) [inline docs/middleware/ics29-fee/end-users.md:17]:
```suggestion
- CLI users who want to manually incentivize IBC packets
```
```suggestion
- CLI Users who want to manually incentivize IBC packets
```

File: docs/middleware/ics29-fee/end-users.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+- Client Developers
```
Comment (crodriguezvega) [inline docs/middleware/ics29-fee/end-users.md:18]:
```suggestion
- Client developers
```
```suggestion
- Client Developers
```

File: docs/middleware/ics29-fee/end-users.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+
+The Fee Middleware module allows end users to add a 'tip' to each IBC packet which will incentivize relayer operators to relay packets between chains. GRPC endpoints are exposed for client developers as well as a simple CLI for manually incentivizing IBC packets.
```
Comment (crodriguezvega) [inline docs/middleware/ics29-fee/end-users.md:21]:
```suggestion
The Fee Middleware module allows end users to add a 'tip' to each IBC packet which will incentivize relayer operators to relay packets between chains. gRPC endpoints are exposed for client developers as well as a simple CLI for manually incentivizing IBC packets.
```
```suggestion
The Fee Middleware module allows end users to add a 'tip' to each IBC packet which will incentivize relayer operators to relay packets between chains. GRPC endpoints are exposed for client developers as well as a simple CLI for manually incentivizing IBC packets.
```

File: docs/middleware/ics29-fee/end-users.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+<!--
+order: 6
+-->
+
+# For End Users
+
+Learn how to incentivize IBC packets using the ICS29-Fee module. {synopsis}
```
Comment (crodriguezvega) [inline docs/middleware/ics29-fee/end-users.md:7]:
```suggestion
Learn how to incentivize IBC packets using the ICS29 Fee Middleware module. {synopsis}
```
```suggestion
Learn how to incentivize IBC packets using the ICS29-Fee module. {synopsis}
```

File: docs/middleware/ics29-fee/end-users.md (hunk 1 above)
Comment (crodriguezvega) [inline docs/middleware/ics29-fee/end-users.md:25]:
```suggestion
For an in depth guide on how to use the ICS29 Fee Middleware module using the CLI please take a look at the [wiki](https://github.com/cosmos/ibc-go/wiki/Fee-enabled-fungible-token-transfers#asynchronous-incentivization-of-a-fungible-token-transfer) on the `ibc-go` repo.
```
```suggestion
For an in depth guide on how to use the ICS29 Fee module using the CLI please take a look at the [wiki](https://github.com/cosmos/ibc-go/wiki/Fee-enabled-fungible-token-transfers#asynchronous-incentivization-of-a-fungible-token-transfer) on the `ibc-go` repo.
```

File: docs/middleware/ics29-fee/end-users.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+- Client Developers
+
+
+The Fee Middleware module allows end users to add a 'tip' to each IBC packet which will incentivize relayer operators to relay packets between chains. GRPC endpoints are exposed for client developers as well as a simple CLI for manually incentivizing IBC packets.
+
+## CLI Users
+
+For an in depth guide on how to use the ICS29 Fee module using the CLI please take a look at the [wiki](https://github.com/cosmos/ibc-go/wiki/Fee-enabled-fungible-token-transfers#asynchronous-incentivization-of-a-fungible-token-transfer) on the `ibc-go` repo.
+
+## Client Developers
```
Comment (crodriguezvega) [inline docs/middleware/ics29-fee/end-users.md:27]:
```suggestion
## Client developers
```
```suggestion
## Client Developers
```

**Review by crodriguezvega — APPROVED** [review]
Thanks for this one, @seantking! Left lots of nits :)

**Comment by seantking** [comment]
> Thanks for this one, @seantking! Left lots of nits :)

Nice review!

**Review by damiannolan — APPROVED** [review]
Nice work, LGTM once @charleenfei's docs are linked! 🙌

**Pull request merged**